src/jackdaw/test.clj

```diff
@@ -67,7 +67,7 @@
 (defn consume-all-outputs!
```

Comment:
We replaced this test namespace with [the test machine](https://github.com/FundingCircle/jackdaw/pull/28) (apologies for the breaking change). There will be a conflict on this file when you update the master branch. You can resolve by simply accepting the upstream branch.

Reply:
+1